src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -30,6 +31,43 @@ namespace System.Diagnostics
     /// </summary>
     public partial class Activity
     {
+        private const int RequestIdMaxLength = 1024;
+
+        // Used to generate an ID it represents the machine and process we are in.  
+        private static readonly string s_uniqSuffix = "-" + GetRandomNumber().ToString("x") + ".";
+
+        //A unique number inside the appdomain, randomized between appdomains. 
+        //Int gives enough randomization and keeps hex-encoded s_currentRootId 8 chars long for most applications
+        private static long s_currentRootId = (uint)GetRandomNumber();
```

Comment:
readonly?

Reply:
Done

Reply:
Ah, is used in Interlocked.Increment in a different file